src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -97,18 +97,22 @@ typedef enum {
 } EnHorseAnimationIndex;
 
 typedef enum {
-    /* 0 */ HORSE_EPONA,
-    /* 1 */ HORSE_HNI,
-    /* 2 */ HORSE_2,
-    /* 3 */ HORSE_3,
-    /* 4 */ HORSE_4
+    /* 0 */ HORSE_TYPE_EPONA,
+    /* 1 */ HORSE_TYPE_HNI,
+    /* 2 */ HORSE_TYPE_2,
+    /* 3 */ HORSE_TYPE_BANDIT,
+    /* 4 */ HORSE_TYPE_DONKEY // Cremia's donkey
 } HorseType;
 
-#define ENHORSE_GET_2000(thisx) ((thisx)->params & 0x2000)
-#define ENHORSE_GET_4000(thisx) ((thisx)->params & 0x4000)
-#define ENHORSE_GET_8000(thisx) ((thisx)->params & 0x8000)
+#define ENHORSE_PARAMTYPE_BANDIT 0x2000
+#define ENHORSE_PARAMTYPE_4000 0x4000
+#define ENHORSE_PARAMTYPE_DONKEY 0x8000
 
-enum {
+#define ENHORSE_PARAMTYPE_ISBANDIT(thisx) ((thisx)->params & ENHORSE_PARAMTYPE_BANDIT)
+#define ENHORSE_PARAMTYPE_IS4000(thisx) ((thisx)->params & ENHORSE_PARAMTYPE_4000)
+#define ENHORSE_PARAMTYPE_ISDONKEY(thisx) ((thisx)->params & ENHORSE_PARAMTYPE_DONKEY)
```

Comment:
For these I don't think we really need to split params into separate subsystems. Perhaps something like `IS_BANDIT_TYPE` would be better?

Reply:
Same for `ENHORSE_PARAMTYPE_BANDIT` and the others?

Reply:
Yeah